src/app/item-page/field-components/metadata-values/metadata-values.component.ts

```diff
@@ -55,6 +55,8 @@ export class MetadataValuesComponent implements OnChanges {
 
   @Input() browseDefinition?: BrowseDefinition;
 
+  @Input() img?: string;
```

Comment:
Please add a comment to describe this optional input (similar to others above this).  Should say something like "An optional image to display next to the metadata value (e.g. for external identifiers)"